docsource/modules150-160.rst

```diff
@@ -686,7 +686,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | sale_crm                                        | Nothing to do        |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| sale_expense                                    |                      |                                                 |
+| sale_expense                                    | Nothing to do        |                                                 |
```

Comment:
@diamondnguyenkc có file noupdate_changes.xml thì phải có migration để update dữ liệu cho các record no_update="1" nhé

Reply:
vâng ạ

Reply:
@diamondnguyenkc module này có migration, không sửa Nothing to do à em?

Reply:
vâng để e sửa lại ạ